src/phantom.js

```diff
@@ -69,34 +89,34 @@ export default class Phantom {
                 this.commands.delete(command.id);
             } else if (message.indexOf('<event>') === 0) {
                 let json = message.substr(7);
-                logger.debug('Parsing: %s', json);
+                this.logger.debug('Parsing: %s', json);
                 const event = JSON.parse(json);
 
                 var emitter = this.events[event.target];
                 if (emitter) {
                     emitter.emit.apply(emitter, [event.type].concat(event.args));
                 }
             } else {
-                logger.info(message);
+                this.logger.info(message);
             }
         });
 
-        this.process.stderr.on('data', data => logger.error(data.toString('utf8')));
-        this.process.on('exit', code => logger.debug(`Child exited with code {${code}}`));
+        this.process.stderr.on('data', data => this.logger.error(data.toString('utf8')));
```

Comment:
Yes, I think that providing a custom phantom path that would do something like `echo "test" 1>&2` and then catching the output would be possible (Also I tested it now and it printed "test" as expected, with node 4.4.7).

However the syntax used in test files is not really familiar to me... I'll have a closer look at it later.